.github/workflows/ci_cron.yml

```diff
@@ -328,3 +328,18 @@ jobs:
       - name: Annotate errors
         if: ${{ failure() }}
         uses: ./.github/actions/annotate
+
+  gem_notifications:
+    runs-on: ubuntu-20.04
+    env:
+      gems: "activerecord bunny dalli delayed_job excon http httpclient mongo puma sidekiq sinatra tilt rack rails rake redis resque unicorn"
```

Comment:
Could this accept an array argument? It might be nicer to maintain something like: 
```yaml
  env:
    gems:
      - activerecord
      - bunny
      - dalli
      - etc... 
```

Reply:
I will explore this. I agree that looks cleaner!

Reply:
I've tried a couple of different formats, but none that GA likes. Maybe we can look at this together? I'm sure there's something...